server/models.py

```diff
@@ -313,10 +314,9 @@ class Assignment(Model):
     files = db.Column(JsonBlob)  # JSON object mapping filenames to contents
     course = db.relationship("Course", backref="assignments")
 
-
-    UserAssignment = namedtuple('UserAssignment',
-                                ['assignment', 'subm_time', 'group', 'final_subm'])
-
+    user_assignment = namedtuple('UserAssignment',
```

Comment:
I just wanted to make QuantifiedCode happy - I'm ambivalent about the naming 

https://www.quantifiedcode.com/app/project/e6ae811f0c244655b5026e0883ffb8f9/diff/b6b6de97aab18bc2f5a731d8d647395dec2ddf00/9b6eba5fae560e577b645209fd8e5a2f0309b26e/issues?groups=code_patterns%3A3ueqPTsn